src/util/ConfigManager/ConfigManager.cpp

```diff
@@ -80,49 +80,40 @@ std::string ConfigManager::createJsonPointerString(
 }
 
 // ____________________________________________________________________________
-void ConfigManager::verifyPathToConfigOption(
-    const std::vector<std::string>& pathToOption,
-    std::string_view optionName) const {
+void ConfigManager::verifyPath(const std::vector<std::string>& path) const {
   // We need at least a name in the path.
-  if (pathToOption.empty()) {
+  if (path.empty()) {
     throw std::runtime_error(
-        "The vector 'pathToOption' is empty, which is not allowed. We need at "
-        "least a name for a working path to a configuration option.");
+        "The vector 'path' is empty, which is not allowed. We need at least a "
+        "name for a working path to a configuration option, or manager.");
```

Comment:
Suggestion:
```
"It is forbidden to call `addCofigOption` with an empty vector as the first argument, because we need a name for the added option.
```

Reply:
Done, see [f0be725](https://github.com/ad-freiburg/qlever/pull/1058/commits/f0be725ad0de4e336677f3ca45a5ff16bc8c89b4).